src/Microsoft.AspNetCore.Mvc.Testing/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -33,14 +43,20 @@
     </ItemGroup>
 
     <ItemGroup>
-      <AssemblyAttribute 
-        Condition=" '%(WebApplicationFactoryContentRootAttribute.Identity)' != '' "
-        Include="Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactoryContentRootAttribute">
-        <_Parameter1>%(WebApplicationFactoryContentRootAttribute.AssemblyName)</_Parameter1>
-        <_Parameter2>%(WebApplicationFactoryContentRootAttribute.ContentRootPath)</_Parameter2>
-        <_Parameter3>%(WebApplicationFactoryContentRootAttribute.ContentRootTest)</_Parameter3>
-        <_Parameter4>%(WebApplicationFactoryContentRootAttribute.Priority)</_Parameter4>
-      </AssemblyAttribute>
+      <AttributeLine
+        Condition="'@(WebApplicationFactoryContentRootAttribute)' != ''"
+        Include="@(WebApplicationFactoryContentRootAttribute->'[assembly: Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactoryContentRoot(&quot;%(AssemblyName)&quot;, @&quot;%(ContentRootPath)&quot;, @&quot;%(ContentRootTest)&quot;, &quot;%(Priority)&quot;)]')" />
```

Comment:
You can't do this. This means the package stops working in F#. Just use `WriteCodeFragment` since it does the right thing